<ui version="4.0" >
 <class>PaintToolbox</class>
 <widget class="QWidget" name="PaintToolbox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vertex Painting</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>2</number>
   </property>
   <item row="4" column="0" colspan="2" >
    <widget class="QFrame" name="pick_frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QComboBox" name="pick_mode" >
        <item>
         <property name="text" >
          <string>Nearest vertex color</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Pixel color</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_pick" >
        <property name="text" >
         <string>Picking mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QFrame" name="pen_extra_frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" colspan="2" >
       <widget class="QCheckBox" name="backface_culling" >
        <property name="text" >
         <string>backfaces</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_pen" >
        <property name="text" >
         <string>Pen</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Paint on:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2" >
       <widget class="QComboBox" name="pen_type" >
        <item>
         <property name="text" >
          <string>pixel</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>percentual of bbox diagonal</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="pen_radius" >
        <property name="decimals" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <double>70.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>0.100000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QCheckBox" name="invisible_painting" >
        <property name="text" >
         <string>hidden polygons</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QFrame" name="pen_frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="deck_box" >
        <property name="toolTip" >
         <string>Change the opacity of the color (this is not the opengl alpha value)</string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_deck" >
        <property name="text" >
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_penmodus" >
        <property name="text" >
         <string>Modus</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="QComboBox" name="pen_modus" >
        <item>
         <property name="text" >
          <string>Normal</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="deck_slider" >
        <property name="toolTip" >
         <string>Change the opacity of the color (this is not the opengl alpha value)</string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="color_frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="ColorWid" native="1" name="back" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="ColorWid" native="1" name="front" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="autoFillBackground" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="set_bw" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>10</y>
        <width>25</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="meshlab.qrc" >:/images/bw.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="switch_me" >
      <property name="geometry" >
       <rect>
        <x>100</x>
        <y>40</y>
        <width>25</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="meshlab.qrc" >:/images/switch.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="autoRaise" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QFrame" name="frame_3" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QFrame" name="top_frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>70</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="fill_button" >
      <property name="geometry" >
       <rect>
        <x>60</x>
        <y>10</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Fill an area</string>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="meshlab.qrc" >:/images/fill.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
      <property name="autoExclusive" >
       <bool>true</bool>
      </property>
      <property name="autoRaise" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="pick_button" >
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>10</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Pick a color</string>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="icon" >
       <iconset resource="meshlab.qrc" >:/images/pick.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
      <property name="autoExclusive" >
       <bool>true</bool>
      </property>
      <property name="autoRaise" >
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="pen_button" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>10</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string>Paint with the pen</string>
      </property>
      <property name="text" >
       <string>...</string>
      </property>
      <property name="icon" >
       <iconset resource="meshlab.qrc" >:/images/pinsel2.png</iconset>
      </property>
      <property name="iconSize" >
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="checkable" >
       <bool>true</bool>
      </property>
      <property name="autoExclusive" >
       <bool>true</bool>
      </property>
      <property name="autoRaise" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWid</class>
   <extends>QWidget</extends>
   <header>colorwid.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="meshlab.qrc" />
 </resources>
 <connections/>
</ui>
